src/index.ts

```diff
@@ -137,4 +145,44 @@ export default class BugzillaAPI {
 
     return this.searchBugs(searchParams);
   }
+
+  public async getComment(commentId: number) {
+    let comment = await this.link.get(
+      `bug/comment/${commentId}`,
+      object(CommentsSpec),
+    );
+
+    if (!comment) {
+      throw new Error(`Failed to get comment #${commentId}.`);
+    }
+
+    return Object.fromEntries(comment.comments);
+  }
+
+  public async getComments(bugId: number) {
+    let comments = await this.link.get(
+      `bug/${bugId}/comment`,
+      object(CommentsSpec),
+    );
+
+    if (!comments) {
+      throw new Error(`Failed to get comments of bug #${bugId}.`);
+    }
+
+    return Object.fromEntries(comments.bugs);
+  }
+
+  public async createComment(bugId: number, content: CreateCommentContent) {
```

Comment:
I'm guessing the vast majority of the time callers only care about passing the comment string. What do you think to changing the method to look like this:

```suggestion
  public async createComment(bugId: number, comment: string, options: Partial<Omit<CreateCommentContent, "comment">> = {}) {
    let content = {
      comment,
      is_private: options.is_private ?? false,
    }
```

Reply:
Great suggestion, thanks! Done.